test/support/repo.ex

```diff
@@ -0,0 +1,3 @@
+defmodule GuardianOneTime.Repo do
```

Comment:
This should match the other namespace don't you think?

Reply:
No, this is completely isolated to this package's test support.